### ISSUE TITLE
Retroarch 1.7.6 installed via snap without audio

### ISSUE THREAD

**Issue by Wellvergton** [libretro/retroarch-snap#20]
## Description
Retroarch without audio in every core, no matter the configuration and no matter the driver.

### Expected behavior
Play the audio.

### Actual behavior
Every core runs without audio, the configuration is correct but there is no sound. As I installed this through snap I reverted to the previous version and the audio is fine. Returning to the current version and running through terminal in verbose mode shows the following message:

[ERROR] failed_to_start_audio_driver

### Steps to reproduce the bug
1. Open the retroarch
2. Run some game

### Version/Commit
* RetroArch: 1.7.6/315d962

### Environment information
* OS: Ubuntu 16.04.5

**Comment by futwick** [comment]
Same problem on Ubuntu 18.04.1

**Comment by orbea** [comment]
@Wellvergton Is this now fixed? https://github.com/libretro/retroarch-snap/issues/21#issuecomment-461521823

**Comment by Wellvergton** [comment]
Yes. Bug fixed. But I want to report another bug: at the fisrt time that a launched the new revision the program started with the frame counter enabled, I disable this, reconfigure the inputs and when I started a game the configurations was lost, the input became the default and the frame counter was enabled again. I had to configure everything again and save the configuration manualy. Should I open a new issue or is ok report this by here?

**Comment by orbea** [comment]
@Wellvergton I would make a new issue so more people see it. I am not sure if its a snap or RetroArch issue, maybe can you rephrase your issue and make it clear how to reproduce it step by step too?

**Comment by Wellvergton** [comment]
After refresh the snaps packages I started the Retroarch. The first thing I noted was the frame counter on the corner, because was not enable in the previous revision, after this I tried to use the Dual Shock 3 and the inputs was not working, so I configured the inputs and disabled the frame counter and when I started a game the frame counter was in the corner again and the inputs lost the configuration. I had to reconfigure everything and save manualy the configuration. After this I had no more problems.
